Wikipedia/Code/ShareActivityController.swift

```diff
@@ -0,0 +1,94 @@
+@objc(WMFShareActivityController)
+class ShareActivityController: UIActivityViewController {
+    
+    init(url: URL, userDataStore: MWKDataStore, context: AnalyticsContextProviding) {
+        let article = userDataStore.fetchArticle(with: url)
+        var items = [Any]()
+        let text: String
+        
+        if let article = article {
+            if let title = article.displayTitle {
+                text = "\"\(title)\" on @Wikipedia"
```

Comment:
The declaration for text can be moved here since it's only used within the scope of this if statement.